game/core/src/main/java/com/gg/game/model/spells/Spell.java

```diff
@@ -0,0 +1,108 @@
+package com.gg.game.model.spells;
+
+import com.gg.game.model.Drawable;
+import com.gg.game.model.characters.Character;
+import com.gg.game.model.utils.Point;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Spell implements Castable, Drawable {
+    private final float range;
+    private final float duration;
+    float activeTime = 0;
+    Character activeSpellCaster;
+    final List<Character> affected;
+    private boolean active;
+
+    static final float DEFAUL_RANGE = 1;
```

Comment:
```suggestion
    static final float DEFAULT_RANGE = 1;
```
Refactor to catch other misspellings.